src/common/util.ts

```diff
@@ -20,7 +20,7 @@ export function executeFile(
   stderr: string
 }> {
   return new Promise((resolve, reject) => {
-    const fpath = VscUri.parse(textDocument.uri).path
+    const fpath = VscUri.parse(textDocument.uri).fsPath
```

Comment:
Documentation seems to recommend using `fsPath` when working with a file.

https://github.com/microsoft/vscode-uri/blob/95e03c06f87d38f25eda1ae3c343fe5b7eec3f52/src/index.ts#L646-L648

Reply:
Thanks for the notice